cli/src/wallet.rs

```diff
@@ -65,7 +65,7 @@ impl WalletSubCommands for App<'_, '_> {
                         .index(1)
                         .value_name("ACCOUNT_ADDRESS")
                         .required(true),
-                    "Account key URI. "
+                    "Account."
```

Comment:
nit: Similar to other comment, I think the expanded text is a bit awkward:
```
Account. Address is one of:
...
```
Looking at the text for `balance` subcommand, maybe something like this
```
Account contents to show.
```
which will expand to:
```
Account contents to show. Address is one of:
...
```